src/main/java/org/broadinstitute/consent/http/models/DataAccessRequest.java

```diff
@@ -40,7 +44,13 @@ public class DataAccessRequest {
   public DataAccessRequestData data;
 
   @JsonProperty
-  public Boolean draft;
+  public Boolean draft = true;
+
+  @JsonProperty
+  public Boolean expired = false;
+
+  @JsonProperty
+  public long expiresAt = -1;
```

Comment:
I would prefer to not store computed data in a POJO. Could you instead annotate the getters for these values with `@JsonProperty`? Storing data here means that you need to recompute when their source data changes.

Or, the UI client could also compute these values, as that's the only use of them right now.

Reply:
I went down this path and had to revert my changes.

https://github.com/google/gson/issues/232

We've configured [consent+Jersey ](https://github.com/DataBiosphere/consent/blob/acbe32328afec051cbc2f25f22aa0bc74572afdc/src/main/java/org/broadinstitute/consent/http/ConsentApplication.java#L192)to use Gson.  Gson doesn't (easily) allow us to implement this.